Posts/2021/02/Get-Services, PowerShell 7 VS Windows PowerShell 5.1

```diff
@@ -0,0 +1,150 @@
+---
+post_title: Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+username: farisnt@gmail.com
+Catagories: PowerShell, Windows Services
+Summary: Show how to get services information using both Windows PowerShell 5.1 and PowerShell 7
+---
+
+# Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+
+**Q**: How can I get the Username, and StartType for a Windows Service?
```

Comment:
Do you mean StartType or StartUpType in PowerShell 7>

Reply:
Both values exist as a member of the `Get-Service` command. The StartUpType output is similar to StartType, but the only difference between them is the StartType will only return the StartType=Automatic, even if the service is Automatic Delayed.
I chose StartType instead of StartUpType just to reduce the return result text size.
Do you like to make it StartUpType instead of StartType?